shared/utils/urls.ts

```diff
@@ -55,7 +56,7 @@ export function isUrl(text: string) {
 
   try {
     const url = new URL(text);
-    return url.hostname !== "";
+    return url.hostname !== "" && !BLACKLIST_PROTOCOLS.includes(url.protocol);
```

Comment:
```suggestion
    const blockedProtocols = [
      "javascript:",
      "file:",
      "vbscript:",
      "data:",
    ]

    return url.hostname !== "" && !blockedProtocols.includes(url.protocol);
```

Please just inline the protocols, no need to be in the separate file as it isn't used elsewhere. Otherwise this PR looks good  now 🙏

Reply:
Changes have been comitted